frontend/ui/widget/qrmessage.lua

```diff
@@ -62,15 +62,18 @@ function QRMessage:init()
 
     local image_widget = QRWidget:new{
         text = self.text,
-        width = .9 * self.width,
-        height = .9 * self.height,
+        width = self.width and .9 * self.width,
+        height = self.height and .9 * self.height,
```

Comment:
I think we all like our `0` in `0.9` :) (I don't remember I've seen any `.9` in our code.)

Reply:
(That, and, also, rounding ;p).

Reply:
I'll have to be careful, because I'm very used to this way.